### PR TITLE
Work and Aobut section created

### PR DESCRIPTION
**Pull request by Nuri1977**
## Work and About section 

- Languages, frameworks, and skills section added (HTML and CSS )
- Icons section created
- Work section with cards components created
-  Background object added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,30 +4,153 @@
...
+        <a href="@" class="btn btn-card">See Project </a>
+      </div>
+      <div class="card">
+        <img src="app_sceenshot.jpg" alt="mypage" />
+				<h3 class="card-header">Multi-Post stories</h3>
+				<p class="card-paragraph">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          Doloribus, nam. Sint ut quisquam nihil voluptas.
+          Fugit omnis exercitationem dolorum praesentium sint nemo ea sequi, dolores facere molestias excepturi vero reiciendis.
+				</p>
```
Comment (andersonlebon) [inline index.html]: - [ ] One of the requirements says that all projects in my work section should be in the `<ul>` tag like this:
```
 <ul class="card-container">
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
      <li class="card"></li>
</ul>
``` 
- [ ] the work container should also use the `grid` CSS property.

**Review by andersonlebon — CHANGES_REQUESTED** [review]
Hi @

Good job so far! I know you have worked hard on making this milestone But there are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- You  are following GitHub actions :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your design is looking great, Good job :clap: :+1: :100: :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Nuri1977) [reply index.html]: Thank you, Anderson. Please check out my fixed commits and give me further instructions on what to do?

**Review by lily-coder — APPROVED** [review]
Hi @Nuri1977 , 

Good job implementing changes as requested by the reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**